/**********************************************************/
//
// File:
//
// Purpose:
//
// Project:
//
/**********************************************************/




try {
		
}
catch (const std::filesystem::filesystem_error& e) {
	output_fse(e);
}
catch (...) {
	output_em(unknown_exception_caught_pkg);
	std::cout << "unknown exception caught...\n";
}